src/main/java/httpclient/helpers/URIBuilderHelper.java

```diff
@@ -0,0 +1,31 @@
+package httpclient.helpers;
+
+import org.apache.hc.core5.net.URIBuilder;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import static helpers.ConfigSetUp.*;
+
+public class URIBuilderHelper {
+
+    private static URIBuilder getUriBuilder(String endpoint) {
```

Comment:
Why not return URI instead of URIBuilder?

Reply:
united 2 methods in 1